libs/src/services/solanaWeb3Manager/index.js

```diff
@@ -120,6 +120,7 @@ class SolanaWeb3Manager {
    * Creates a solana bank account from the web3 provider's eth address
    */
   async createUserBank () {
+    if (!this.web3Manager) throw new Error('No web3Manager supplied to solanaWeb3Manager')
```

Comment:
Wasn't clear from the code - is there a reason we would allow you to init the `solanaWeb3Manger` without a `web3Manger`, and then require these checks here?

Reply:
will clear this up, good callout. the protocol dashboard has no web3 manager b/c it has no POA interaction, but we do need solana interaction. so some methods on the solana web3 manager need to do a check. could abstract this a bit and certainly comment